crates/biome_cli/tests/snapshots/main_commands_init/creates_config_file.snap

```diff
@@ -26,22 +26,28 @@ Welcome to Biome! Let's get you started...
 
 Files created ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  - biome.json: Your project configuration. Documentation: https://biomejs.dev/reference/configuration
+  - biome.json
+    Your project configuration. See https://biomejs.dev/reference/configuration
 
 Next Steps ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   1. Setup an editor extension
-     Get live errors as you type and format when you save. Learn more: https://biomejs.dev/guides/getting-started#editor-setup
+     Get live errors as you type and format when you save.
+     Learn more: https://biomejs.dev/guides/integrate-in-editor/
 
   2. Try a command
-     biome ci checks for lint errors and verifies formatting. Run biome --help for a full list of commands and options.
+     biome check verifies formatting and lints.
+     biome --help displays the available commands.
 
-  3. Read the documentation
-     Our website serves as a comprehensive source of guides and documentation: https://biomejs.dev
+  3. Migrate from ESLint and Prettier
+     biome migrate eslint attempts to migrate your ESLint configuration to Biome.
+     biome migrate prettier attempts to migrate your Prettier configuration to Biome.
```

Comment:
We should have a resource (website page) that explains this better. A guide, maybe. Not a blocker but something we should prepare, I think.

Reply:
Yeah I was thinking the sane.